source/winAPI/displayTracking.py

```diff
@@ -10,33 +10,69 @@
 and we notify the user of changes to the orientation.
 """
 
+from ctypes import windll
 from dataclasses import dataclass
 import enum
 from typing import (
 	Optional,
 )
 
+from logHandler import log
 import ui
 import winUser
 
+from .constants import SystemMetrics
+
 
 class Orientation(enum.Enum):
-	NOT_INITIALIZED = enum.auto()
 	PORTRAIT = enum.auto()
 	LANDSCAPE = enum.auto()
 
 
 @dataclass
 class OrientationState:
-	width: int = 0
-	height: int = 0
-	style: Orientation = Orientation.NOT_INITIALIZED
+	width: int
+	height: int
+	style: Orientation
+
+
+_orientationState: Optional[OrientationState] = None
+
+
+def initialize():

```

Comment:
Can this module be excluded from the add-on API? Given this is new code, and we don't know there is a demand for it within the API, keeping it excluded will let us iterate safely on the design.

Reply:
This also goes for the other new modules that don't need to be exposed.